### PR TITLE
Feat/algolia

### PR DESCRIPTION
**Pull request by patricksimonian**
## Summary
Serves as the main integration branch for the algolia feature

### CODE REVIEW

File: .jenkins/openshift/deploy-master.yaml [hunk 1]
```diff
@@ -41,8 +41,8 @@ objects:
   kind: PersistentVolumeClaim
   metadata:
     annotations:
-      volume.beta.kubernetes.io/storage-class: gluster-file
-      volume.beta.kubernetes.io/storage-provisioner: kubernetes.io/glusterfs
+      volume.beta.kubernetes.io/storage-class: netapp-file-standard
+      volume.beta.kubernetes.io/storage-provisioner: kubernetes.io/netapp
```
Comment (ShellyXueHan) [inline .jenkins/openshift/deploy-master.yaml]: no need to specify the `storageClassName`?

File: .pipeline/lib/deploy.js [hunk 2]
```diff
@@ -23,19 +23,21 @@ const getParamsByEnv = (env, pr) => {
...
       return params;
     case ENVS.TEST:
-      return { ...params, SSO_BASE_URL_VALUE: 'https://sso-test.pathfinder.gov.bc.ca' };
+      return { ...params, SSO_BASE_URL_VALUE: 'https://sso-test.pathfinder.gov.bc.ca', ALGOLIA_INDEX_NAME_SUFFIX: 'test' };
     case ENVS.DEV:
       return {
         ...params,
         SSO_BASE_URL_VALUE: 'https://sso-dev.pathfinder.gov.bc.ca',
         SSO_CLIENT_ID_VALUE: `devhub-web-${pr}`,
+        ALGOLIA_INDEX_NAME_SUFFIX: `-build-${pr}`,
```
Comment (ShellyXueHan) [inline .pipeline/lib/deploy.js]: build in dev?

File: .jenkins/openshift/deploy-master.yaml (hunk 1 above)
Comment (patricksimonian) [reply .jenkins/openshift/deploy-master.yaml]: we actually moved to ephemeral jenkins so this is not even needed anymore

File: .pipeline/lib/deploy.js (hunk 2 above)
Comment (patricksimonian) [reply .pipeline/lib/deploy.js]: no builds, its just the name of the suffix. this is the same suffix used by the dev image. may as well make them same same

**Pull request merged**